src/main/java/org/springframework/integration/aws/outbound/KplMessageHandler.java

```diff
@@ -403,7 +451,8 @@ private PutRecordRequest buildPutRecordRequest(Message<?> message) {
 			if (!StringUtils.hasText(partitionKey) && this.partitionKeyExpression != null) {
 				partitionKey = this.partitionKeyExpression.getValue(getEvaluationContext(), message, String.class);
 			}
-			Assert.state(partitionKey != null, "'partitionKey' must not be null for sending a Kinesis record. "
+			Assert.state(partitionKey != null,
+					"'partitionKey' must not be null for sending a Kinesis record. "
```

Comment:
This change doesn't look reasonable.

Reply:
The line length was crossing beyond 120, at-least I could fix them to reduce some of the debt.